File: uri/1 - Basics/1046/Main.java
import java.util.*;

class Main{
	public static void main(String [] args){
	    Scanner scanner = new Scanner(System.in);
		int a = scanner.nextInt();
		int b = scanner.nextInt();	    
	    
	    if(a >= b)
	        System.out.printf("O JOGO DUROU %d HORA(S)\n", (24 - a) + b);
	    else
	        System.out.printf("O JOGO DUROU %d HORA(S)\n", b - a);

	}
}

File: uri/1 - Basics/1046/Main.go
package main

import "fmt"

func main() {
	var a, b int

	fmt.Scanf("%d %d", &a, &b)

	if a >= b {
		fmt.Printf("O JOGO DUROU %d HORA(S)\n", (24-a)+b)
	} else {
		fmt.Printf("O JOGO DUROU %d HORA(S)\n", b-a)
	}

}
